### PR TITLE
Feat_T31342: adjust design: STN modal window - diplanbau

### PR DESCRIPTION
**Pull request by sakutademos**
**Ticket:** https://yaits.demos-deutschland.de/T31035

**Description:** This PR adds new classes to some elements in the `StatementModal` component which makes it possible to adjust required changes in the DiPlanBau Project.

_It is connected with the same PR in the DiPlanBau project._

- [x] Link all relevant tickets
- [x] Move the tickets on the board accordingly

### CODE REVIEW

File: client/js/components/statement/publicStatementModal/StatementModal.vue [hunk 1]
```diff
@@ -121,28 +121,30 @@
...
-          :toolbar-items="{
-            mark: true,
-            strikethrough: true
-          }"
-          ref="statementEditor"
-          :required="formData.r_isNegativeReport !== '1'"
-          :value="formData.r_text || ''"
-          @input="val => setStatementData({r_text: val})" />
+        <div :class="prefixClass('c-statement__text')">
+          <label
```
Comment (hwiem) [inline client/js/components/statement/publicStatementModal/StatementModal.vue:125]: Can't we use DpLabel here?

Comment (sakutademos) [reply client/js/components/statement/publicStatementModal/StatementModal.vue]: yes, we can! Thx! done here https://github.com/demos-europe/demosplan-core/pull/856/commits/0045c6fd3722a52f73754ee530331cc4be13651f

File: client/js/components/statement/publicStatementModal/StatementModal.vue [hunk 2]
```diff
@@ -121,28 +121,23 @@
...
-          ref="statementEditor"
-          :required="formData.r_isNegativeReport !== '1'"
-          :value="formData.r_text || ''"
-          @input="val => setStatementData({r_text: val})" />
+        <div :class="prefixClass('c-statement__text')">
+          <dp-label
+            :text="Translator.trans('statement.detail.form.statement_text')"
+            for="statementText" />
+          <dp-editor
+            class="u-mb"
```
Comment (spiess-demos) [inline client/js/components/statement/publicStatementModal/StatementModal.vue:129]: Please use the `prefixClass` function in the public area as long as we do not automate that (which we can't as we use twig here).

```suggestion
            :class="prefixClass('u-mb')"
```

**Review by spiess-demos — CHANGES_REQUESTED** [review]

File: client/js/components/statement/publicStatementModal/StatementModal.vue [hunk 3]
```diff
@@ -121,28 +121,23 @@
...
-            strikethrough: true
-          }"
-          ref="statementEditor"
-          :required="formData.r_isNegativeReport !== '1'"
-          :value="formData.r_text || ''"
-          @input="val => setStatementData({r_text: val})" />
+        <div :class="prefixClass('c-statement__text')">
+          <dp-label
+            :text="Translator.trans('statement.detail.form.statement_text')"
+            for="statementText" />
```
Comment (hwiem) [inline client/js/components/statement/publicStatementModal/StatementModal.vue:127]:
```suggestion
            for="statementText"
            :required="formData.r_isNegativeReport !== '1'" />
```

Comment (sakutademos) [reply client/js/components/statement/publicStatementModal/StatementModal.vue]: ach.. thx! :))

**Review by hwiem — APPROVED** [review]
lgtm, code review only

**Review by spiess-demos — APPROVED** [review]
lgtm! (code review only)

**Pull request merged**